### DISCUSSION TITLE
Batch Scan: Convert JPG to OCR PDFs

### DISCUSSION THREAD

**Discussion by mrbgfromv** [cyanfish/naps2#136] [Support]
Hi there,

I've been using naps2 for quite some time now and I have to say that this peace of software works just fine.
Today I tried the Batch-Scan function and it gave me perfect jpgs - just as promised.
What I really wanted was a set of PDF files with their text processed via OCR.
Now what do you suggest is the best way to convert the JPGs to PDFs?
Thanks in advance!

**Comment by cyanfish** [comment]
Put a .pdf extension on the filename and it will create pdfs. If you have OCR configured it should include that.

**Reply by mrbgfromv** [reply]
This works like a charm. Thank you very much!
